diesel_cli/src/main.rs

```diff
@@ -194,7 +194,19 @@ fn run_infer_schema(matches: &ArgMatches) {
     let database_url = database::database_url(matches);
     let schema_name = matches.value_of("schema");
 
-    let filtering_tables = matches.values_of("table-name").map(|v| v.collect())
+    let filtering_tables = matches.values_of("table-name").map(|v| {
```

Comment:
What does `v` stand for here and in the outer scope? Value? I [agree](https://scribbles.pascalhertleif.de/programming-wisdom.html) that it should have a short name, but it's not immediately obvious what this is. Even after looking at the [clap docs](https://docs.rs/clap/2.20.0/clap/struct.ArgMatches.html#method.values_of) I only got the vague `clap::Values` type, which I assume can be seen as an `Iterator<Item=String>` here.

Can you maybe swap out that `.map.unwrap_or` for a `if let … else`? This way it's a bit clearer we are dealing with an `Option` here and, hopefully, that the inner `map` is on an Iterator.